streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala

```diff
@@ -129,6 +130,44 @@ class ReceiverSuite extends TestSuiteBase with Timeouts with Serializable {
     }
   }
 
+  test("check the state of `Receiver` and `ReceiverSupervisor` when receivers are killed") {
```

Comment:
I'd add `(SPARK-5205)` to the name of this test, if it will fit, or just add a comment on the next line referencing the JIRA number.